ts/Series/PackedBubble/PackedBubbleIntegration.ts

```diff
@@ -35,31 +35,46 @@ import VerletIntegration from '../Networkgraph/VerletIntegration.js';
  */
 function barycenter(this: PackedBubbleLayout): void {
     const layout = this,
-        gravitationalConstant = layout.options.gravitationalConstant,
+        gravitationalConstant = layout.options.gravitationalConstant || 0,
         box = layout.box,
         nodes = layout.nodes as Array<PackedBubblePoint>;
 
     let centerX: number,
         centerY: number;
 
     for (const node of nodes) {
-        if (layout.options.splitSeries && !node.isParentNode) {
-            centerX = (node.series.parentNode as any).plotX;
-            centerY = (node.series.parentNode as any).plotY;
-        } else {
-            centerX = box.width / 2;
-            centerY = box.height / 2;
-        }
         if (!node.fixedPosition) {
-            (node.plotX as any) -=
-                ((node.plotX as any) - (centerX as any)) *
-                (gravitationalConstant as any) /
+            const series = node.series,
+                parentNode = series.parentNode;
+
+            if (layout.options.splitSeries && parentNode) {
+                centerX = parentNode.plotX || 0;
+                centerY = parentNode.plotY || 0;
+            } else {
+                centerX = box.width / 2;
+                centerY = box.height / 2;
+            }
+
+            (node.plotX as number) -=
+                ((node.plotX as number) - centerX) *
+                gravitationalConstant /
                 (node.mass * Math.sqrt(nodes.length));
 
-            (node.plotY as any) -=
-                ((node.plotY as any) - (centerY as any)) *
-                (gravitationalConstant as any) /
+            (node.plotY as number) -=
```

Comment:
We usually try to avoid casting, as casting a `number|undefined` to `number` is arguably as bad as casting to any. Instead, we typically do `(node.plotY || 0)` even though it's more verbose in the built code.